plugins/org.locationtech.udig.tutorials.preferredzoomlevels/src/org/locationtech/udig/tutorials/preferredzoomlevels/View.java

```diff
@@ -69,6 +70,11 @@ public IStatusLineManager getStatusLineManager() {
         return getViewSite().getActionBars().getStatusLineManager();
     }
 
+    @Override
```

Comment:
like in other tutorials, a mapSite instance should be created in createPartControl and used right here to access it:

```mapSite = new MapSite(getViewSite(), this);```

Reply:
The whole getter is unneccessary as I had another look. This `View.java` extends the `DefaultMapViewPart` which is doing exactly what you are requesting. Therefore I removed this line here and in the other file.